spec/feature/sessions_spec.rb

```diff
@@ -1,59 +1,52 @@
-require_relative "../integration_test_helper"
 require "gds_api/test_helpers/account_api"
 require "gds_api/test_helpers/email_alert_api"
 require "gds_api/test_helpers/content_store"
 
-class SessionsTest < ActionDispatch::IntegrationTest
+RSpec.describe "Sessions", type: :feature do
   include GdsApi::TestHelpers::AccountApi
   include GdsApi::TestHelpers::EmailAlertApi
   include GovukPersonalisation::TestHelpers::Features
 
+  before do
+    stub_content_store_has_item("/")
+  end
+
   context "Given a signing in user" do
-    should "Log the user in and redirect them to manage their account" do
+    it "Logs the user in and redirect them to manage their account" do
       given_a_successful_login_attempt
-      get new_govuk_session_callback_path(code: "code", state: "state")
-      # initial redirect to /account/home which redirects on to the manage page
-      assert_response :redirect
-      follow_redirect!
-      assert response.location = GovukPersonalisation::Urls.manage
+      visit(new_govuk_session_callback_path(code: "code", state: "state"))
+
+      expect(page.current_url).to eq("#{GovukPersonalisation::Urls.one_login_your_services}/")
```

Comment:
Similar to a previous comment, we're not checking for the redirect anymore. Is that just because it seemed overkill?

Reply:
Checking for the current path in a system test is fine (it will fail if there isn't a redirect, so it does both jobs)